content/opt/run

```diff
@@ -19,6 +19,10 @@ chown -R rundeck:rundeck /var/rundeck
 chown -R rundeck:adm /var/log/rundeck
 chown -R rundeck:rundeck /var/lib/rundeck
 chown -R rundeck:rundeck /opt/rundeck-defaults
+rm -rf /tmp/rundeck
```

Comment:
These lines were added to overcome the permission issue when GID and UID are passed as env variables.